.github/workflows/config.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       matrix:
         stack: [heroku-18, heroku-20, heroku-22]
-        redis_version: ["", "4", "5", "6", "6.0.17"]
+        redis_version: ["", "4", "5", "6", "6.0.17", "7.0.9"]
```

Comment:
I think you just want "7" here. We have the specific 6.0 version because "6" maps to "6.2".